docs/upgrade/2.1.0_to_2.2.0.rst

```diff
@@ -0,0 +1,114 @@
+the *SecureDrop Workstation Updater* will alert you to workstation updates. You
+must have `configured an administrator password <https://tails.boum.org/doc/first_steps/welcome_screen/administration_password/>`_
+on the Tails welcome screen in order to use the graphical updater.
+
+Perform the update to 2.2.0 by clicking "Update Now":
+
+.. image:: ../images/securedrop-updater.png
+
+Performing a manual update
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+If the graphical updater fails and you want to perform a manual update instead,
+first delete the graphical updater's temporary flag file, if it exists (the
+``.`` before ``securedrop`` is not a typo): ::
+
+  rm ~/Persistent/.securedrop/securedrop_update.flag
+
+This will prevent the graphical updater from attempting to re-apply the failed
+update and has no bearing on future updates. You can now perform a manual
+update by running the following commands: ::
+
+  cd ~/Persistent/securedrop
+  git fetch --tags
+  gpg --keyserver hkps://keys.openpgp.org --recv-key \
+   "2359 E653 8C06 13E6 5295 5E6C 188E DD3B 7B22 E6A3"
+  git tag -v 2.2.0
+
+The output should include the following two lines: ::
+
+    gpg:                using RSA key 2359E6538C0613E652955E6C188EDD3B7B22E6A3
+    gpg: Good signature from "SecureDrop Release Signing Key <securedrop-release-key-2021@freedom.press>" [unknown]
+
+
+Please verify that each character of the fingerprint above matches what is
+on the screen of your workstation. If it does, you can check out the
+new release: ::
+
+    git checkout 2.2.0
+
+.. important:: If you do see the warning "refname '2.2.0' is ambiguous" in the
+  output, we recommend that you contact us immediately at securedrop@freedom.press
+  (`GPG encrypted <https://securedrop.org/sites/default/files/fpf-email.asc>`__).
+
+Finally, run the following commands: ::
+
+  ./securedrop-admin setup
+  ./securedrop-admin tailsconfig
+
+Updating Tails
+--------------
+Follow the graphical prompts to update to the latest version of the Tails
+operating system on your *Admin* and *Journalist Workstations*.
+
+.. important ::
+
+   Older versions of Tails had problems with automatic updates, which SecureDrop
+   tries to correct automatically. Check the version of Tails on your *Admin* and
+   *Journalist Workstations* (**Applications ▸ Tails ▸ About Tails**). If you are
+   running a version older than Tails 4.23, and did not receive an automatic
+   upgrade prompt after connecting to the Internet, perform a
+   :ref:`manual update <Update Tails Manually>`. If this also fails, please
+   don't hesitate to contact us.
+
+Troubleshooting Kernel Issues
+-----------------------------
+SecureDrop 2.2.0 includes a kernel update on the *Application* and *Monitor
+Servers*, from version 5.4.136 to version 5.15.18. As with all kernel updates,
+we have extensively tested this update against
+:ref:`recommended hardware <Specific Hardware Recommendations>`.
```

Comment:
Assuming we move forward with the Mac Mini deprecation, we could insert another note to that effect here as well.

Reply:
Done.